src/presence_analyzer/utils.py

```diff
@@ -1,40 +1,38 @@
 # -*- coding: utf-8 -*-
-"""
-Helper functions used in views.
-"""
+"""Helper functions used in views."""
 
+# Standard libraries
 import csv
-from json import dumps
-from functools import wraps
 from datetime import datetime
+from functools import wraps
+from json import dumps
+import logging
```

Comment:
Move import of whole modules on top level and add one blank line to separate import types